resources/lib/playerinfo.py

```diff
@@ -85,18 +99,13 @@ def onPlayBackSeek(self, time, seekOffset):  # pylint: disable=invalid-name
         log(3, '[PlayerInfo %d] Event onPlayBackSeek time=%d offset=%d' % (self.thread_id, time, seekOffset))
         self.last_pos = time // 1000
 
-        # If we seek beyond the end, quit Player
-        if self.last_pos >= self.total:
```

Comment:
So the reason to have this, was to ensure that seeking until the end would effectively end the current stream.

I have seen the following behaviour in the past:
- You seek to the end (e.g. to mark the video as watched)
- It would jump back to the last position and continue playing

Reply:
Okay, I'll readd this just to be sure.

Reply:
No, let's leave it out. We can put it back if it is needed.